src/components/YourFeed.tsx

```diff
@@ -40,13 +41,15 @@ export interface DispatchProps {
     onDeletePost: (post: Post) => void;
     onSavePost: (post: Post) => void;
     onSharePost: (post: Post) => void;
+    onUnfollowFeed: (feedUrl: string) => void;
 }
 
 export interface StateProps {
     navigation: any;
     posts: Post[];
     settings: Settings;
     displayFeedHeader: boolean;
+    showUnfollow: boolean;
```

Comment:
isn't this redundant at the moment?

Reply:
No, this is for hiding the unfollow option for your own feed.